examples/provider/provider.tf

```diff
@@ -1,4 +1,4 @@
 provider "project" {
-  url = "${var.artifactory_url}/projects"
+  url = "${var.artifactory_url}"
```

Comment:
Do you want to allow override url?

Reply:
I don't understand how this would affect url override. User can set `var.artifactory_url` to whatever they want. The `/projects` would constraint them further, no?

Reply:
Are you referring to the fact we parse this `url` and then only use the host part for Resty?

Reply:
[example of a need to override](https://github.com/jfrog/terraform-provider-artifactory/issues/85)